<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemToolSNWriteWidget</class>
 <widget class="QWidget" name="SystemToolSNWriteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>407</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>532</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5320</width>
    <height>3300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>532</width>
       <height>330</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>532</width>
       <height>330</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 253, 255);
border-radius: 4px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>56</number>
      </property>
      <property name="rightMargin">
       <number>56</number>
      </property>
      <property name="bottomMargin">
       <number>32</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTitle">
          <property name="styleSheet">
           <string notr="true">border:none;
font-size: 22px;
font-weight: 500;
color: #2D3440;</string>
          </property>
          <property name="text">
           <string>SN码写入</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="labelControllCabinetSN">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 16px;
font-weight: 350;
color: #2D3440;</string>
            </property>
            <property name="text">
             <string>控制柜SN码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditControllCabinetSN">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-color:#FAFDFF;
border: 1px solid #DFE3E5;
border-radius: 4px;
color: #2D3440;
padding-left: 12px;
padding-right: 12px;

font-size: 14px;
font-weight: 350;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="labelRobotArmSN">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 16px;
font-weight: 350;
color: #2D3440;</string>
            </property>
            <property name="text">
             <string>机械臂SN码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditRobotArmSN">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-color:#FAFDFF;
border: 1px solid #DFE3E5;
border-radius: 4px;
color: #2D3440;
padding-left: 12px;
padding-right: 12px;

font-size: 14px;
font-weight: 350;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="labelErrorMessage">
            <property name="styleSheet">
             <string notr="true">    color:#CE4949;</string>
            </property>
            <property name="text">
             <string>机械臂SN码不能为空值，重新输入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>44</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>44</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 3px;
background: #FAFDFF;
border: 1px solid #DFE3E5;
font-size: 16px;
font-weight: normal;
color: #2D3440;</string>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnWirteABind">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>44</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>44</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
  width: 196px;
  height: 44px;
  background: #0047BB;
  border-radius: 4px;
  color: #FAFDFF;
}
QPushButton::disabled{
  background: rgb(55, 71, 95);
}</string>
            </property>
            <property name="text">
             <string>写入并绑定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
